src/libsync/syncfileitem.h

```diff
@@ -284,6 +284,8 @@ class OWNCLOUDSYNC_EXPORT SyncFileItem
     bool _isRestoration BITFIELD(1); // The original operation was forbidden, and this is a restoration
     bool _isSelectiveSync BITFIELD(1); // The file is removed or ignored because it is in the selective sync list
     EncryptionStatus _e2eEncryptionStatus = EncryptionStatus::NotEncrypted; // The file is E2EE or the content of the directory should be E2EE
+    EncryptionStatus _e2eEncryptionMaximumAvailableStatus = EncryptionStatus::NotEncrypted;
+    EncryptionStatus _e2eEncryptionStatusRemote = EncryptionStatus::NotEncrypted;
```

Comment:
can you now have a file changing from encrypted to unencrypted dynamically during sync ?

Reply:
@mgallien during the migration, the encryption status (you chose the name status) can become of higher value due to migration from older e2ee to newer (from 1.2 to 2.0 for example)

Reply:
OK